internal/voting/vote_test.go

```diff
@@ -1,9 +1,10 @@
-//go:build pglive
+// go:build pglive
```

Comment:
Can revert.  BTW, you and @charithabandi do this frequently, and I'm not clear on why.  You can set VSCode test tags if that's the reason.

```json
    "go.testTags": "ext_test,pglive",
    "go.buildTags": "ext_test,pglive",
```

Reply:
Woah mind blown

Reply:
This is still not updated. remove the extra space?

Reply:
Lol I fixed and unfixed sorry